graph/src/data/store/mod.rs

```diff
@@ -1,7 +1,12 @@
 use graphql_parser::query;
 use graphql_parser::schema;
 
+use bigdecimal::BigDecimal;
+use diesel::pg::Pg;
```

Comment:
I don't like seeing a dependency on Diesel in the `graph` crate. @leodasvacas @fordN I assume the only alternative would be to create `SqlValue(Value)` ad `SqlEntity(Entity)` newtypes in the `graph-store-postgres` crate and wrap entities in those before creating SQL statements?

Reply:
That's the only alternative I see. I think it's ok for `graph` to take the dependency, but ideally we'd re-export  diesel from `graph` and then `graph-store-postgres` would not depend directly on diesel, so we can control the diesel version and features from a single `Cargo.toml`. This is what was done for `slog` and `tokio`.

Reply:
That's ok but we will very likely replace Postgres at some point and to avoid making that too hard, depending on it anywhere else but in `store/postgres` is a bad idea.